articles/ros2_threat_model.md

```diff
@@ -0,0 +1,3291 @@
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">3</td>
+    <td class="danger">3</td>
+    <td class="warning">2</td>
+    <td class="danger">3</td>
+    <td class="danger">3</td>
+    <td>14</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="success">✓</td>
+    <td class="warning">▲</td>
+    <td class="success">✓</td>
+    <td class="success">✓</td>
+    <td class="warning">▲</td>
+    <td class="warning">▲</td>
+    <td>
+      <ul>
+        <li>Deployment service should be authenticated• Communication with
+the deployment service should be done over a secure channel.</li>
+      </ul>
+    </td>
+    <td> </td>
+  </tr>
+
+  <tr>
+    <td>An attacker intercepts the binaries sent by the depoyment service.</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="success">1</td>
+    <td class="danger">3</td>
+    <td class="warning">2</td>
+    <td class="danger">3</td>
+    <td class="danger">3</td>
+    <td>12</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td>
+      <ul>
+        <li>Deployment service should be authenticated• Communication with
+the deployment service should be done over a secure channel.</li>
+      </ul>
+    </td>
+    <td> </td>
+  </tr>
+
+  <tr>
+    <td>An attacker prevents the robot and the deployment service from
+communicating.</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="success">1</td>
+    <td class="danger">3</td>
+    <td class="warning">2</td>
+    <td class="danger">3</td>
+    <td class="danger">3</td>
+    <td>12</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="success">✓</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    <td class="danger">✘</td>
+    
```

Comment:
Recommend rephrasing: "The application considered in this section is tele-operation of a Turtlebot 3"